release/golden_notebook_tests/workloads/torch_tune_serve_test.py

```diff
@@ -85,8 +85,34 @@ def train_mnist(test_mode=False, num_workers=1, use_gpu=False):
         checkpoint_at_end=True)
 
 
-def get_best_model(best_model_checkpoint_path):
-    model_state = torch.load(best_model_checkpoint_path)
+def get_remote_model(remote_model_checkpoint_path):
+    address = os.environ.get("RAY_ADDRESS")
+    if address is not None and address.startswith("anyscale://"):
```

Comment:
Would this also work with the Ray OSS client?

Reply:
Yep, for OSS this would go to the `else` block and use `ray.remote` to fetch the model. I updated the code with some better path isolation and successfully ran this script on a Ray cluster!